frontend/packages/console-app/src/components/cloud-shell/CloudShellExec.tsx

```diff
@@ -156,7 +158,7 @@ const CloudShellExec: React.FC<CloudShellExecProps> = ({
         setWsOpen(false);
 
         // Check the Cloud Shell to see if it has any hints as to why the terminal connection was closed
-        const cloudShellCR = getCloudShellCR(workspaceName, namespace);
+        const cloudShellCR = getCloudShellCR(workspaceModel, workspaceName, namespace);
```

Comment:
Please add `workspaceModel` to the dependency array. It's present below the return statement. line 208

Reply:
Thanks. Fixed this as well as other linter errors.